src/controller/CHIPDeviceController.cpp

```diff
@@ -104,18 +105,18 @@ CHIP_ERROR DeviceController::Init(NodeId localDeviceId, PersistentStorageDelegat
 
     if (systemLayer != nullptr && inetLayer != nullptr)
     {
-        mSystemLayer = systemLayer;
-        mInetLayer   = inetLayer;
+        mSystemLayer          = systemLayer;
+        mInetLayer            = inetLayer;
+        mBleManagerConfigured = true;
     }
     else
     {
 #if CONFIG_DEVICE_LAYER
```

Comment:
I think this entire 'Init chip stack and ble' has no place inside the controller. It was already bad, we are making it worse by adding more functionality.

Let native callers have a stack initialization - they are fully capable of doing so.